### PR TITLE
LPS-136367 Adding new test case for 7.3.10.1 version in SAMLUpgrade.testcase

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LPS-136367

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro [hunk 1]
```diff
@@ -816,6 +816,35 @@ definition {
 		BlogsEntry.viewStatus(entryStatus = "Approved");
 	}
 
+	@description = Create certificates again after upgrade
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro:819]: It seems " is missing around the description text.

File: portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro [hunk 2]
```diff
@@ -816,6 +816,35 @@ definition {
...
+	macro configureCertificatesAfterUpgrade {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro:820]: If you name this macro as "createCertificatedAfterUpgrade" then you can delete the @description part.

Comment (weller0246) [reply portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro:820]: Changed naming to your idea, the meaning of the macro is more clear with this choice

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM!

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/upgrades/SAMLUpgrade.testcase [hunk 3]
```diff
@@ -31,6 +31,21 @@ definition {
...
+	test ViewSAMLConfigurationAfterUpgrade73101 {
+		property data.archive.type = "data-archive-saml";
+		property database.types = "db2,mariadb,mysql,postgresql,sqlserver,sybase";
+		property portal.version = "7.3.10.1";
+
+		Upgrade.viewSAMLGeneralConfiguration(samlRoleType = "Identity Provider");
+
+		Upgrade.viewSAMLSPConnectionConfiguration();
+
+		Upgrade.configureCertificatesAfterUpgrade();
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/upgrades/SAMLUpgrade.testcase:44]: This macro does not exist in the Upgrade.macro, please check the macro names you added in the previous commit:
https://github.com/gaborlovasdotcom/liferay-portal/pull/133/commits/4401bce9fbcd6b1635717ab4cf016456a7417c65

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
Continued in https://github.com/gaborlovasdotcom/liferay-portal/pull/136

**Pull request closed without merging**